source/process/training.rst

```diff
@@ -632,3 +632,20 @@ Example of finished Mattermost Avatar:
 
 .. image:: https://cloud.githubusercontent.com/assets/177788/25364270/0425b738-2916-11e7-9a23-5ced2d9dfc8f.png
 
+---------------------------------------------------------
+Mattermost Teams and Naming Conventions
+---------------------------------------------------------
+
+Staff must use their full name for their username as in `first.last` on the community server.  Staff are also encouraged to link to their github account using the command `/github connect` so your github handle will appear on your profile popover card.  All Mattermost core staff should be issued a OneLogin account with a username of `first.last`.  If issued a OneLogin account you should switch your sign-in method from email/password to OneLogin via the `Account Settings > Security > Sign-in Method` page.  Once this is done your username will be set from SAML.
+
+Contributors team
+---------------------------------------------------------
+This is a team open for any and all to join.  It is the main place where core staff, contributors, users, customers and partners interact.  Our goal is to do as much work as possible in the open.  If you find yourself creating a private channel on this team you should pause and ask yourself if it really needs to be private.  Mattermost staff will be sync'ed to the appropriate channels based on their LDAP group sync settings.
+
+Mattermost Staff team
```

Comment:
Wondering about the term "colleagues" or "staff" instead of Mattermost Staff, given we have limited letters in the team name